lib/blueprinter/base.rb

```diff
@@ -10,18 +10,74 @@
 
 module Blueprinter
   class Base
+    # Specify a field or method name used as an identifier. Usually, this is
+    # something like :id
```

Comment:
Add: `Note: identifiers are always rendered and considered their own view, similar to the :default view`

Reply:
It's not showing here but I added it to this https://github.com/procore/blueprinter/pull/29/commits/c14c797aae8f0431834611fd24b0d58c1c2ac6ea